### ISSUE TITLE
BUILD file not found in directory 'modfile' of external repository @org_golang_x_mod. Add a BUILD file to a directory to mark it as a package. and referenced by '@bazel_gazelle//language/go:go'

### ISSUE THREAD

**Issue by filmil** [filmil/bazoekt#1]
For some reason, gazelle actions stop here.

```
╰─>$ bazel run //:gazelle-update-repos
ERROR: /home/filmil/.cache/bazel/_bazel_filmil/75cdad6b91268382b93c303491cc2de5/external/bazel_gazelle/language/go/BUILD.bazel:15:11: no such package '@org_golang_x_mod//modfile': BUILD file not found in directory 'modfile' of external repository @org_golang_x_mod. Add a BUILD file to a directory to mark it as a package. and referenced by '@bazel_gazelle//language/go:go'
ERROR: Analysis of target '//:gazelle-update-repos' failed; build aborted: 
INFO: Elapsed time: 0.168s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)
ERROR: Build failed. Not running target
┬─[filmil@instance-3:~/code/bazoekt]─[06:36:13]
│ (g/b:main)
╰─>$ 
```

**Comment by filmil** [comment]
Reported as https://github.com/bazelbuild/bazel-gazelle/issues/1694

**Comment by filmil** [comment]
Solved via #2.
